### ISSUE TITLE
Incorrect minification after quote within comment

### ISSUE THREAD

**Issue by kokizzu** [tdewolff/minify#15]
From [moment.js](http://momentjs.com) 2.8.4 line `842` 
```
            for (i = 0; i < 12; i++) {
                // make the regex if we don't have it already
                mom = moment.utc([2000, i]);
```
result:
```
for (i = 0; i < 12; i++) {
                //make the regex if we don;(function(root,factory){if(typeof define==='function'&&define.amd){define(['moment','jquery','exports'],function(momentjs,$,exports){root.daterangepicker=factory(root,exports,momentjs,$);});}else if(typeof exports!=='undefined'){var momentjs=require('moment');var jQuery;try{jQuery=require('jquery');}catch(err){jQuery=window.jQuery;if(!jQuery)throw new Error('jQuery dependency not found');}
factory(root,exports,momentjs,jQuery);}else
```
comment should be removed..

**Comment by tdewolff** [comment]
The problem actually occurs at line 875:
`this._ordinalParseLenient = new RegExp(this._ordinalParse.source + '|' + /\d{1,2}/.source);`

The inline regexp at the end is not interpreted as such, because whether it is a divisor or regexp when encountering a `/` depends on the parser state. But this is just a tokenizer, so I use the rules JSMin uses for determining whether it's one or the other. Apparently the stackoverflow post didn't copy the rules correctly..

**Comment by tdewolff** [comment]
Fixed in 5c13cb5a03e579f4edd12d64227dafb28a449c3a
